packages/mineral-ui-tokens/tokens/input/index.json

```diff
@@ -8,7 +8,11 @@
   },
   "props": [
     { "name": "input_backgroundColor",          "value": "{!white}" },
-    { "name": "input_backgroundColor_disabled", "value": "{!gray_20}" },
-    { "name": "input_color_placeholder",        "value": "{!gray_60}" }
+    { "name": "input_backgroundColor_disabled", "value": "{!gray_10}" },
+    { "name": "input_color_placeholder",        "value": "{!gray_60}" },
+    { "name": "input_size_small",               "value": "1.5em" },
```

Comment:
let's use the same convention I mentioned above: sm, md, lg, etc.

Reply:
This follows size_ convention